plugins/catalog-backend/src/ingestion/LocationAnalyzer.ts

```diff
@@ -40,7 +40,7 @@ export class RepoLocationAnalyzer implements LocationAnalyzer {
         // Probably won't handle properly self-hosted git providers with custom url
         annotations: { [`${source}/project-slug`]: `${owner}/${name}` },
       },
-      spec: { type: 'other', lifecycle: 'unknown' },
+      spec: { type: 'other', lifecycle: 'unknown', owner: 'guest' },
```

Comment:
I realize `guest` is used in the examples, but considering the purpose of this is to publish real PR's, the term `guest` seems a little out of place imo. I would probably prefer something that implies it's not complete. What about `unknown` or `tbd`, etc.?

Reply:
This is a fair comment. `unknown` is more informative for users who see the value in the catalog.

If someone uses the guest login mechanism, and an entity is owned by `guest`, does Backstage still treat the entity as if it is owned by the logged in guest user? It used to do that but the functionality may have changed.